common/predictive-text/build-bundler.js

```diff
@@ -1,41 +1,18 @@
-/*
- * Note:  while this file is not meant to exist long-term, it provides a nice
- * low-level proof-of-concept for esbuild bundling of the various Web submodules.
- *
- * Add some extra code at the end of src/index.ts and run it to verify successful bundling!
- */
-
 import esbuild from 'esbuild';
 
-
 await esbuild.build({
   alias: {
     'tslib': '@keymanapp/tslib'
   },
   bundle: true,
-  sourcemap: true,
   format: "esm",
-  nodePaths: ['../../node_modules'],
+  sourcemap: true,
+  target: "es5",
+  external: ['fs', 'vm'],
```

Comment:
Would it make sense to sort the properties alphabetically for consistency across all build-bundlers? Makes it easier to compare them

Reply:
I'll be worrying about this when centralizing the bundling configs.